include/experimental/__p0009_bits/extents.hpp

```diff
@@ -16,11 +16,14 @@
 
 #pragma once
 #include "dynamic_extent.hpp"
+#include "fwd.hpp"
 
 #ifdef __cpp_lib_span
 #include <span>
 #endif
 #include <array>
+#include <type_traits>
+#include <stdexcept>
```

Comment:
Did anything in this header throw an `std::exception` subclass before?

Reply:
> Would you consider just calling std::abort as before? Alternately, would you consider adding a configurable precondition violation handler (e.g., a macro that takes a message and just calls std::abort by default)?

Sure that sounds like a better approach. I'm also wondering if `extents.hpp` is the right location for the `terminate_if_invalid_strides` function. Maybe `layout_stride.hpp` is a better place?